virtcontainers/container.go

```diff
@@ -300,7 +300,7 @@ func (c *Container) createContainersDirs() error {
 func (c *Container) mountSharedDirMounts(hostSharedDir, guestSharedDir string) ([]Mount, error) {
 	var sharedDirMounts []Mount
 	for idx, m := range c.mounts {
-		if isSystemMount(m.Destination) || m.Type != "bind" {
+		if m.Type != "bind" {
```

Comment:
This would e.g., bind mount host `/proc` to a sub-directory of the shared dir. I'm not sure that we would want it.

Reply:
@bergwolf well no cause in case of `/proc`, the source type is `proc`. Same thing for `/dev/pts`, the type is `devpts`.
But at the same time we can have something like `/dev/termination-log` where the type is `bind`, which is case where we actually want to bind mount this through the shared dir.

One other way to proceed would be to provide an exhaustive list of the paths we don't want to consider, but this looks more tedious, and we would have to anticipate everything.

WDYT ?

Reply:
yeah, you are right. Thanks for the explanation.

Reply:
No problem ! Glad you agree.